### DISCUSSION TITLE
提升识别效果

### DISCUSSION THREAD

**Discussion by bishengderen** [PaddlePaddle/PaddleOCR#13148] [Q&A]
您好，我在前一段时间使用了本项目的代码，使用的是自动下载的PP-OCRv4 模型。
我在使用中发现了两个问题：
1.有些本应能识别出来的文字却没有识别出来（因为第一行识别出来了，第二行没有），请问怎样提升模型的识别能力呢？或者能换其他效果更好的模型吗？
2，有的照片是横着拍的，这样字是竖着的，这种也没有识别出来，请问该如何设置能识别出来呢？

**Comment by GreatV** [comment]
1. 可以尝试其他模型 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/models_list.md
2. 需要使用一些预处理操作来旋转图片
